pkg/component/gardener/dashboard/deployment.go

```diff
@@ -41,23 +42,29 @@ const (
 	volumeMountPathConfig      = "/etc/gardener-dashboard/config"
 	volumeMountPathLoginConfig = "/app/public/" + dataKeyLoginConfig
 	volumeMountPathAssets      = "/app/public/static/assets"
-	volumeMountPathSession     = "/etc/gardener-dashboard/secrets/session"
-	volumeMountPathOIDC        = "/etc/gardener-dashboard/secrets/oidc"
-	volumeMountPathGithub      = "/etc/gardener-dashboard/secrets/github"
 
-	volumeNameConfig       = "gardener-dashboard-config"
-	volumeNameLoginConfig  = "gardener-dashboard-login-config"
-	volumeNameConfigAssets = "gardener-dashboard-assets"
-	volumeNameSession      = "gardener-dashboard-sessionsecret"
-	volumeNameOIDC         = "gardener-dashboard-oidc"
-	volumeNameGithub       = "gardener-dashboard-github"
+	volumeSubPathSession           = "sessionSecret"
+	volumeMountPathSession         = "/etc/gardener-dashboard/secrets/session/sessionSecret"
```

Comment:
Let's avoid this mixedCase in volume mounts. Use `-` instead
```suggestion
	volumeMountPathSession         = "/etc/gardener-dashboard/secrets/session/session-secret"
```

or even

```suggestion
	volumeMountPathSession         = "/etc/gardener-dashboard/secrets/session/secret"
```

Reply:
If we want to change this we need to adapt the implementation in g/dashboard first and make a release there.